spmd/tensor/api.py

```diff
@@ -114,25 +115,19 @@ def wrap(e, mesh, placements):
             return rs
 
     @classmethod
-    # pyre-fixme[3]: Return type must be annotated.
     def from_local(
         cls,
-        # pyre-fixme[2]: Parameter must be annotated.
-        local_tensor,
-        # pyre-fixme[2]: Parameter must be annotated.
-        device_mesh=None,
-        # pyre-fixme[2]: Parameter must be annotated.
-        placements=None,
-        # pyre-fixme[2]: Parameter must be annotated.
-        run_check=True,
-    ):
+        local_tensor: torch.Tensor,
+        device_mesh: DeviceMesh = None,
+        placements: List[Placement] = None,
+        run_check: bool = True,
```

Comment:
run_check is still not implemented, right?

Reply:
Yeah sure let me add a docstr, `run_check` is actually partially implemented by broadcasting for replicated tensor, we still need to check additional attributes like tensor sizes, dtype, etc.